app/views/home/index.html.erb

```diff
@@ -1,11 +1,10 @@
-<nav>
-  <%= link_to 'Login' %>
-  <%= link_to 'Register with Spotify' %>
-  <%= link_to 'About' %>
-</nav>
 
-<section class='about'>
+<section id='about'>
   <p>
     At parties, there is often one person who ends up being in charge of the music. What if everyone’s preferences could be taken into consideration by simply inviting one’s friends via a text message, from there, the music would then take care of itself. DJ Spoti aggregates the favorite songs of all session party users and cultivates a playlist with all user preferences in mind.
   </p>
 </section>
+
+<section id='login-register'>
```

Comment:
I was thinking that we'd have login at the top of the page with about below, @Mackenzie-Frey, @manojpanta, what are your thoughts?